utilities/test_suite/HOST/README_AUDIO.md

```diff
@@ -0,0 +1,51 @@
+# AMD ROCm Performance Primitives (RPP) Test Suite
+The AMD ROCm Performance Primitives (RPP) audio test suite can be used to validate the functionality and performance of the AMD ROCm Performance Primitives (RPP) audio augmentations.
+-   HOST backend - (On a CPU with HOST backend)
+
+## Command Line Arguments
+The test suite accepts the following command line arguments:
+-   input_path: The path to the input folder. Default is $cwd/../TEST_AUDIO_FILES/eight_samples_single_channel_src1
+-   case_start: The starting case number for the test range (0-0). Default is 0
+-   case_end: The ending case number for the test range (0-0). Default is 0
+-   test_type: The type of test to run (0 = Unit tests, 1 = Performance tests). Default is 0
+-   case_list: A list of specific case numbers to run. Must be used in conjunction with --test_type
+-   qa_mode: Outputs of audio augmentations from tests will be compared with golden outputs - (0 / 1). Default is 0
+-   num_runs: Specifies the number of runs for running the performance tests
+-   preserve_output: preserves the output or performance logs generated from the previous test suite run - (0 = remove output or performance logs / 1 = preserve output or performance logs). Default is 1
+-   batch_size: Specifies the batch size to use for running tests. Default is 1
+
+## Running the Tests for HOST Backend
+The test suite can be run with the following command:
+python runAudioTests.py --input_path <input_path> --case_start <case_start> --case_end <case_end> --test_type <test_type>
+
+## Modes of operation
+-   QA mode - Tolerance based PASS/FAIL tests for RPP AUDIO HOST functionalities checking match between output and preset golden outputs. Please note that QA mode is only supported with a batch size of 8.
+``` python
+python runAudioTests.py --case_start 0 --case_end 0 --test_type 0 --qa_mode 1 --batch_size 8
+```
+-   Unit test mode - Unit tests allowing users to pass a path to a folder containing audio files, to execute the desired functionality and variant once, report RPP execution wall time, save and view output
```

Comment:
Please remove the save and view output, since for audio, there is no output that is gonna be generated

Reply:
Done